tests/login.js

```diff
@@ -17,7 +17,7 @@ export const productTitle = Selector('.inventory_item_name');
 export const firstProductAZ = 'Sauce Labs Backpack';
```

Comment:
You need to import the page models inside the test page, so you can use the page methods. Dont' forget that. it should look something like this: 

import ProductsPage from "./pages/productsPage";